indico/modules/rb_new/client/js/modules/roomList/filters/EquipmentForm.jsx

```diff
@@ -94,8 +103,27 @@ export default class EquipmentForm extends FilterFormComponent {
                                            this.setFeature(feat.name, checked);
                                        }} />
                     ))}
-                    {!!availableFeatures.length && !!availableEquipment.length && <br />}
-                    {availableEquipment.map(equip => (
+                    {!!availableFeatures.length && !!availableEquipment.length && (
+                        <>
+                            <Accordion styleName="equipment-accordion">
+                                <Accordion.Title active={showEquipment} index={0} onClick={this.handleClick}>
+                                    <Icon name="dropdown" />
+                                    <Translate>See detailed equipment</Translate>
+                                </Accordion.Title>
+                                <Accordion.Content active={showEquipment}>
+                                    {availableEquipment.map(equip => (
+                                        <Form.Checkbox checked={equipment.includes(equip)}
+                                                       key={equip}
+                                                       label={equip}
+                                                       onChange={(__, {checked}) => {
+                                                           this.setEquipment(equip, checked);
+                                                       }} />
+                                    ))}
+                                </Accordion.Content>
+                            </Accordion>
+                        </ >
```

Comment:
`</>`

Reply:
I removed it now since it wasn't needed after I also removed the `<br>` element